kadai1/watcher041/modules/convExt.go

```diff
@@ -0,0 +1,89 @@
+package modules
```

Comment:
モジュールはパッケージとは別の概念なので別の名前にした方がよさそうです。
「何を」するパッケージなのかで名前をつけるとよいです。

Reply:
ご指摘ありがとうございます。
mainから「拡張子を変換するパッケージ」として以下のように修正しました。
```suggestion
package convExt
```